packages/date/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@wordpress/date",
-	"version": "2.0.1",
+	"version": "2.0.0",
```

Comment:
Why is this being changed?

Reply:
Oops, I think I updated the date package for the `dateAbbreviated` format... but maybe I shouldn't bump this? I always forget how this work.

Reply:
We should update only as part of the package publishing process, and we should never downgrade.

Reply:
Still needs updating.